src/components/WeatherCard/WeatherCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as S from './style';
+
+/* Import Components */
+import TransparentBox from '../TransparentBox';
+import Subtitle from '../Subtitle';
+import Icon from '../Icon';
+
+const WeatherCard = () => {
+  return (
+    <S.Card>
+      <TransparentBox color='lightgray' opacity='0.2'>
+        <Subtitle text='text1' />
```

Comment:
maybe typography is a better name

Reply:
I have 2 typography components: 
1- Title
2- Subtitle

Can i move them to folder with typography name and from them export them with `index,jsx` file?